Chrome/bridge.js

```diff
@@ -10,7 +10,7 @@
 var bridge_error = false;
 var bridge_error_object;
 var xkit_storage = {};
-var bridge_ver = "2.1";
+var bridge_ver = "2.2.1";
```

Comment:
Ah ok I see. 

I also think that this would require a bump in the overall XKit version, in order to deploy. I'm kinda on shaky ground here though so it might be best to wait until @hobinjk has some time and can advise.